src/components/sketches/test3.ts

```diff
@@ -0,0 +1,64 @@
+import p5 from 'p5';
```

Comment:
let's remove this scene and add it later!

Reply:
Got it